packages/customize-widgets/src/components/sidebar-block-editor/style.scss

```diff
@@ -4,10 +4,10 @@
 	z-index: 8;
 }
 
-.customize-control-sidebar_block_editor {
-	margin: 0 ( -$grid-unit-15 ) ( -$grid-unit-15 ) ( -$grid-unit-15 );
-	padding: $grid-unit-15;
-	background: $white;
-	border-top: 1px solid #dcdcde;
-	border-bottom: 1px solid #dcdcde;
+// Style the customizer accordion section with a white background.
+// This selector matches against the first part of the id, which is typically
+// postfixed with a number when there are multiple sidebars.
+#customize-theme-controls *[id^="sub-accordion-section-sidebar-widgets-sidebar"] {
```

Comment:
Do you think adding a class and select it here would be better? ref: https://github.com/WordPress/gutenberg/pull/31487#issuecomment-832518557

I'm fine with both approaches 👍